HDK/Milli5-Arduino CoAP Server/ssni_coap_server/hdlcs.cpp

```diff
@@ -308,7 +308,7 @@ struct mbuf * hdlcs_read(void)
         return r;
     }
     
-    dlog( LOG_DEBUG, "hdlcs_read() - %d", r );
+    //dlog( LOG_DEBUG, "hdlcs_read() - %d", r );

```

Comment:
While debugging, I've also make the same change. But be careful because the documentation here: https://portal7-stage.mitangi.com/milli-developer-kit-reference-application Step #11 shows the logs